src/assets/ajaxcrud.js

```diff
@@ -47,7 +47,7 @@ $(document).ready(function () {
             modal.show();
             modal.setTitle('No selection');
             modal.setContent('You must select item(s) to use this action');
-            modal.addFooterButton("Close", 'btn btn-default', function (button, event) {
+            modal.addFooterButton("Close", 'button','btn btn-default', function (button, event) {
```

Comment:
please can you help me to understand this?

Reply:
There is a funtion addFooterButton which takes 4 argument but only 3 are provided.